plugins/net_plugin/net_plugin.cpp

```diff
@@ -2747,7 +2774,7 @@ namespace eosio {
             boost::system::error_code ec;
             acceptor->cancel( ec );
             acceptor->close( ec );
-         }      
```

Comment:
please avoid whitespace only changes.
same comment for lines 2570-2579

Reply:
I am sorry. My code editor does removing trailing spaces automatically, I never try to configure it otherwise because no one else has ever had problem with it. However, I don't think it makes sense to add them back because it won't preserve the history unless I do a force push which is frowned upon by the team.

Reply:
got it. try to avoid in future, please. it breaks changes history and adds verbosity into PR